app/graphql/types/list_type.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module Types
+  class ListType < Base::Object
+    I18N_PATH = 'graphql.types.list_type'
+
+    graphql_name 'ListType'
+    implements Types::Interfaces::NodeInterface
+    description I18n.t("#{I18N_PATH}.desc")
+
+    field :name, String, null: false, description: I18n.t("#{I18N_PATH}.fields.name")
+
+    field :items,
+          [Types::MovieType],
+          null: true,
+          description: I18n.t("#{I18N_PATH}.fields.items")
+
+    field :deleted_list_id, ID, null: false, description: I18n.t("#{I18N_PATH}.fields.deleted_list_id")
```

Comment:
Please remove deleted_list_id field from ListType. This field is mutation-specific field (used only for delete mutation).
Instead in your operation do ctx['result'] = { deleted_list_id: ctx[:model].id }
This field will be available to the FE when they want to fetch the data for the show page. What will it return to the FE?) They shouldn't be able to fetch it.

Reply:
Thank you, done